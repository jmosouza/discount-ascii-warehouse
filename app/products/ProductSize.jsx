import React from 'react';
import PropTypes from 'prop-types';

const ProductSize = ({ size }) => (
  <p>Size: {size}</p>
);

ProductSize.propTypes = {
  size: PropTypes.number.isRequired,
};

export default ProductSize;
