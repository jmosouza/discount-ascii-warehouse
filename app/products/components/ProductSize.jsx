import React from 'react';
import PropTypes from 'prop-types';

const ProductSize = ({ size }) => (
  <span>Size: {size}</span>
);

ProductSize.propTypes = {
  size: PropTypes.number.isRequired,
};

export default ProductSize;
