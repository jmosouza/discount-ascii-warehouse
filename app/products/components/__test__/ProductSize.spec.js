/* global test, expect */
import renderer from 'react-test-renderer';
import React from 'react';
import ProductSize from '../ProductSize';

test('ProductSize', () => {
  expect(renderer.create(
    <ProductSize
      size={12}
    />,
  )).toMatchSnapshot();
});
