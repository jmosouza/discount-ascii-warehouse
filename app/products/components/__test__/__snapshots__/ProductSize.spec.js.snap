// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductSize 1`] = `
<span>
  Size: 
  12
</span>
`;
